postgres/organisation_service_test.go

```diff
@@ -58,6 +58,18 @@ func TestOrganisationService(t *testing.T) {
 		assert.Equal(t, organisation, organisation2)
 	})
 
+	t.Run("ok: update organisation", func(t *testing.T) {
+		name := tests.Faker.BS()
+		organisation, err := service.CreateOrganisation(ctx, name, nil)
+		tests.Check(err)
+
+		organisation.Name = tests.Faker.BS()
+		organisation2, err := service.UpdateOrganisation(ctx, organisation.ID, organisation.Name)
+		tests.Check(err)
+
+		assert.Equal(t, organisation, organisation2)
```

Comment:
Dit test niet echt dat het geüpdatet is in de databank, enkel dat je jouw input teruggekregen hebt 😛  Je zou nadien nog eens een get moeten doen met hetzelfde id en checken dat dat resultaat hetzelfde is als dat uit jouw update call, zo kan je checken dat het wel degelijk aangepast is in de databank